### PR TITLE
Modularize messaging backend and socket.io server.

### PR DESCRIPTION
**Pull request by jianjunz**
This change makes it possible to add more transport servers in the future.

### CODE REVIEW

File: src/messaging.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
+// Copyright (C) <2015> Intel Corporation
```
Comment (taste1981) [inline src/messaging.js:1]: 2020

File: src/messaging.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+//
+// SPDX-License-Identifier: Apache-2.0
+
+/*
+  This is the backend messaging service for P2P signaling server. It's frontend
```
Comment (taste1981) [inline src/messaging.js:6]: Its frontend

File: src/messaging.js (hunk 1 above)
Comment (jianjunz) [reply src/messaging.js:1]: This file is based on the original `peerserver.js`, so it should still be 2015?

Comment (taste1981) [reply src/messaging.js:1]: this is new file any way. Please set to 2020.

Comment (jianjunz) [reply src/messaging.js]: Done.

File: src/messaging.js (hunk 2 above)
Comment (jianjunz) [reply src/messaging.js]: Done.

**Comment by jianjunz** [comment]
Closing this PR because this change is included in #25 and it was merged.

**Pull request closed without merging**